lapis2/src/main/kotlin/org/genspectrum/lapis/request/CommonSequenceFilters.kt

```diff
@@ -13,9 +13,13 @@ import org.genspectrum.lapis.controller.NUCLEOTIDE_MUTATIONS_PROPERTY
 import org.genspectrum.lapis.controller.OFFSET_PROPERTY
 import org.genspectrum.lapis.controller.ORDER_BY_PROPERTY
 import org.genspectrum.lapis.controller.SPECIAL_REQUEST_PROPERTIES
+import org.springframework.util.MultiValueMap
+
+typealias SequenceFilters = Map<String, List<String>>
+typealias GetRequestSequenceFilters = MultiValueMap<String, String>
```

Comment:
Spring only properly maps GET requests if it is a `MultiValueMap`. Having the full `MultiValueMap` throughout the code was a bit too exhausting and not really necessary, because it's just a thin wrapper around `Map<K, List<V>>`